share/completions/sfdx.fish

```diff
@@ -0,0 +1,1086 @@
+# Tab completion for sfdx (https://developer.salesforce.com/tools/sfdxcli).
+
+function __fish_sfdx_needs_command
```

Comment:
This can be replaced with `__fish_use_subcommand`

Reply:
Thank you for reviewing, I did it!